mod/videos.php

```diff
@@ -358,7 +358,7 @@ function videos_content(App $a) {
 
 	$r = q("SELECT hash, `id`, `filename`, filetype FROM `attach`
 		WHERE `uid` = %d AND filetype LIKE '%%video%%'
-		$sql_extra GROUP BY hash ORDER BY `created` DESC LIMIT %d , %d",
+		$sql_extra GROUP BY hash, `id` ORDER BY `created` DESC LIMIT %d , %d",
```

Comment:
For reference, EXPLAIN output (the version with ANY_VALUE should be the current behaviour before the change):
```
mysql> EXPLAIN SELECT hash, MIN(`id`) AS `id`, MIN(`created`) AS `created`, min(`filename`) AS `filename`, min(`filetype`) as `filetype` FROM `attach`
    -> GROUP BY hash ORDER BY `created` DESC;
+----+-------------+--------+------------+------+---------------+------+---------+------+------+----------+---------------------------------+
| id | select_type | table  | partitions | type | possible_keys | key  | key_len | ref  | rows | filtered | Extra                           |
+----+-------------+--------+------------+------+---------------+------+---------+------+------+----------+---------------------------------+
|  1 | SIMPLE      | attach | NULL       | ALL  | NULL          | NULL | NULL    | NULL |    1 |   100.00 | Using temporary; Using filesort |
+----+-------------+--------+------------+------+---------------+------+---------+------+------+----------+---------------------------------+
1 row in set, 1 warning (0,06 sec)

mysql> EXPLAIN SELECT hash, ANY_VALUE(`id`) AS `id`, ANY_VALUE(`created`) AS `created`, ANY_VALUE(`filename`) AS `filename`, ANY_VALUE(`filetype`) as `filetype` FROM `attach`
    -> GROUP BY hash ORDER BY `created` DESC;
+----+-------------+--------+------------+------+---------------+------+---------+------+------+----------+---------------------------------+
| id | select_type | table  | partitions | type | possible_keys | key  | key_len | ref  | rows | filtered | Extra                           |
+----+-------------+--------+------------+------+---------------+------+---------+------+------+----------+---------------------------------+
|  1 | SIMPLE      | attach | NULL       | ALL  | NULL          | NULL | NULL    | NULL |    1 |   100.00 | Using temporary; Using filesort |
+----+-------------+--------+------------+------+---------------+------+---------+------+------+----------+---------------------------------+
1 row in set, 1 warning (0,00 sec)
```